### PR TITLE
connect to mysql database

### PR DESCRIPTION
**Pull request by KimlengHor**

### CODE REVIEW

File: insert.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var mysql = require('mysql');
```
Comment (dpham123) [inline insert.js]: What is this file being used for?

File: routes/index.js [hunk 2]
```diff
@@ -1,9 +1,27 @@
 var express = require('express');
 var router = express.Router();
+var mysql = require('mysql');
+var app = express()
```
Comment (dpham123) [inline routes/index.js]: What is this variable being used for?

File: routes/index.js [hunk 3]
```diff
@@ -1,9 +1,27 @@
...
+  });
+  
+  connection.connect()
+  
+  connection.query('SELECT * FROM emp', function(err, rows, fields) {
+    if (err) throw err
+    res.render('index', {title: "id: " + rows[0]["id"] + ", " 
+    + "name: " + rows[0]["name"] + ", " 
+    + "age:" + rows[0]["age"]})
+  })
```
Comment (dpham123) [inline routes/index.js]: This works, which is good. Can we improve it to list all the rows and not just row 0? Chances are, the TA will tell us to add something to the DB and see if it shows up on the webpage.

File: insert.js (hunk 1 above)
Comment (KimlengHor) [reply insert.js]: This is needed for retrieving the database. Also, for the connection.

File: routes/index.js (hunk 2 above)
Comment (KimlengHor) [reply routes/index.js]: i should delete the app variable

File: insert.js (hunk 1 above)
Comment (dpham123) [reply insert.js]: I'm talking about the file `insert.js`. Is the file really needed?

Comment (KimlengHor) [reply insert.js]: we don't need it. I was testing on it. I will delete it.

**Pull request merged**